ldap/servers/slapd/pblock_v3.h

```diff
@@ -68,6 +68,7 @@ typedef struct _slapi_pblock_task
     int ldif2db_noattrindexes;
     int ldif_printkey;
     int task_flags;
+    int task_misc;
```

Comment:
When adding new types, it's worth adding them with int32_t types for example rather tahn bare int if possible. this has impacts on future rust ffi